src/app/containers/MediaPlayer/index.test.jsx

```diff
@@ -61,3 +64,16 @@ it('should contain the noscript tag for no-JS scenarios ', () => {
 
   expect(document.querySelector('noscript')).toBeInTheDocument();
 });
+
+it('should log embed source status code when player is loaded', () => {
```

Comment:
Is it worth adding another test for when toggle is disabled?

Reply:
Done.